docs/general/ad-filtering/create-own-filters.md

```diff
@@ -448,30 +448,23 @@ The rule applies only to WebSocket connections.
 >
 > AdGuard for Safari and iOS cannot properly apply rules with `websocket` modifier due to Safari limitations.
 
-#### **`$webrtc`** {#webrtc-modifier}
+#### **`$other`** {#other-modifier}
+
+The rule applies to requests for which the type has not been determined or does not match the types listed above.
+
+#### **`$webrtc` (deprecated)**** {#webrtc-modifier}
 
 > **Deprecation notice**
 >
-> This modifier is deprecated and will be removed in the future. If you need to suppress WebRTC, consider using the `nowebrtc` [scriptlet](#scriptlets).
+> This modifier is deprecated and no longer supported. Rules with it are considered as invalid.
+> If you need to suppress WebRTC, consider using the `nowebrtc` [scriptlet](#scriptlets).
 The rule applies only to WebRTC connections.
```

Comment:
the description may no longer be needed

if not, consider its including into note above or inserting empty line before it

Reply:
I think we should preserve it for history.